ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessorTest.kt

```diff
@@ -97,7 +97,7 @@ class VariableProcessorTest {
 
     @Test
     fun `process variables processes product_name`() {
-        expectVariablesResult("{{ product_name }}", "PRODUCT_NAME")
+        expectVariablesResult("{{ product_name }}", "Annual")
```

Comment:
Just a single test here is good (same as `VariablesTest`).
I added a secondary `PackageVariablesTests` to check more specific details based on each package type.
For this I tested all types in iOS:
```swift
func testProductName() {
    expect(TestData.weeklyPackage.productName) == "Weekly"
    expect(TestData.monthlyPackage.productName) == "Monthly"
    expect(TestData.threeMonthPackage.productName) == "3 months"
    expect(TestData.sixMonthPackage.productName) == "6 months"
    expect(TestData.annualPackage.productName) == "Annual"
    expect(TestData.lifetimePackage.productName) == "Lifetime"
}
```

Which was useful then to also verify localization.

We can add that later but wanted to bring it up so it's on the radar :)

Reply:
Right, we need to add more test data with the different package types. Right now we only have annual, monthly and weekly. But we can do this 👍